libyul/backends/evm/EVMDialect.cpp

```diff
@@ -396,6 +406,69 @@ std::vector<std::optional<BuiltinFunctionForEVM>> createBuiltins(langutil::EVMVe
 				_assembly.appendAuxDataLoadN(static_cast<uint16_t>(literal->value.value()));
 			}
 		));
+
+		builtins.emplace_back(createFunction(
+			"eofcreate",
+			5,
+			1,
+			SideEffects{
+				false,               // movable
+				false,               // movableApartFromEffects
+				false,               // canBeRemoved
+				false,               // canBeRemovedIfNotMSize
+				true,                // cannotLoop
+				SideEffects::Write,  // otherState
+				SideEffects::Write,  // storage
+				SideEffects::Write,  // memory
+				SideEffects::Write   // transientStorage
+			},
```

Comment:
Just like with `ControlFlowSideEffects`, this should be set based on `SemanticInformation`. Fortunately in this case a helper already exists:

```suggestion
			EVMDialect::sideEffectsOfInstruction(evmasm::Instruction::EOFCREATE),
```

Same for `returncontract()`.

Reply:
done